apps/storybook/stories/design/Icon.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta, Canvas, Controls } from "@storybook/addon-docs";
+import * as IconStories from "./Icon.stories";
+
+<Meta of={IconStories} />
+
+# Icon
+
+Vi benytter ikoner som et virkemiddel for hjelpe brukere med å forstå handling eller kontekst. Ikoner kan være med på å skille og fremheve innhold.
+Det er også et fint navigasjonsverkøy som kan vise varsling av feil eller fortelle brukeren om de forlater nettsiden.
+
+Kartverkets designsystem tilbyr et ikonelement som viser og lar deg style ikoner fra
+[Material symbols rounded.](https://fonts.google.com/icons?icon.set=Material+Symbols&icon.style=Rounded)
```

Comment:
pirk: endre litt på setningen så det blir f.eks. "....og lar deg style ikoner fra Google sin Material Symbols Rounded. Gå til [Material symbols rounded sine sider](https://fonts.google.com/icons?icon.set=Material+Symbols&icon.style=Rounded) for å få en oversikt over ikoner og ikonnavn du kan benytte. "

For å gjøre det tydelig at de må til deres nettsider for å finne selve ikonavnet.